doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -79,7 +79,10 @@ Bluetooth® LE
 Bluetooth mesh
 --------------
 
-|no_changes_yet_note|
+* Updated:
```

Comment:
Move the entry under Libraries:

E.g.
Bluetooth libraries and services
--------------------------------

* :ref:`bt_mesh` library:

  * Updated:

    * The :kconfig:option:`BT_MESH_MODEL_SRV_STORE_TIMEOUT` Kconfig option, that is controlling timeout for storing of model states, is replaced by the :kconfig:option:`BT_MESH_STORE_TIMEOUT` Kconfig option.